docs/admin/install.rst

```diff
@@ -0,0 +1,265 @@
+Before setting up the set of VMs used by SecureDrop Workstation, you must configure the Journalist Interface connection and submission key. 
+
+- To add the submission key, run the following command in the ``dom0`` terminal:
+
+  .. code-block:: sh
+
+    sudo cp /tmp/sd-journalist.sec /usr/share/securedrop-workstation-dom0-config/
+
+- Next, create the SecureDrop Workstation configuration file:
+
+  .. code-block:: sh
+   
+    cd /usr/share/securedrop-workstation-dom0-config
+    sudo cp config.json.example config.json
+
+- The ``config.json`` file must be updated with the correct values for your instance. Open it with root privileges in a text editor and update the following fields' values:
+
+  - **submission_key_fpr**: use the value of the submission key fingerprint
+  - **hidserv.hostname**: use the hostname of the Journalist Interface, including the ``.onion`` TLD
+  - **hidserv.key**: use the value of the v2 HidServAuth token for the Journalist Interface, or the v3 private authorization key value if your SecureDrop instance uses v3 onion services
+  - **environment**: use the value ``prod``
+
+.. note:: 
+  You can find the values for the **hidserv.*** fields in the ``/tmp/journalist.txt`` file that you created in ``dom0`` earlier:
+
+  If your instance uses v2 onion services, the file will be formatted as follows:
+
+  .. code-block:: none
+
+    HidServAuth ONIONADDRESS.onion AUTHTOKEN # comments, can be ignored
+
+  If your instance uses v3 onion services, the file will be formatted as follows:
+  
+  .. code-block:: none
+  
+   ONIONADDRESS:descriptor:x25519:AUTHTOKEN
+
+
+- Finally, in the ``dom0`` terminal, run the command:
+
+  .. code-block:: sh
+
+    securedrop-admin --apply
+
+This command will take a considerable amount of time, as it sets up multiple VMs and installs supporting packages. When it completes, your SecureDrop Workstation is finally ready to use!
+
+Test the Workstation
+--------------------
+
+To start the SecureDrop Workstation client, double-click the SecureDrop desktop icon that was set up by the previous command. The preflight updater will start and check for updates. The system should be up-to-date and no updates should be required, but if updates are available follow the instructions in the preflight updater to apply them.
+
+Once the update check is complete, the client will launch. Log in using an existing journalist account and verify that sources are listed and submissions can be downloaded, decrypted, and viewed.
+
+Clean up
+--------
+
+Once you've confirmed that the client is working correctly, delete the files containing the key and Journalist Interface details from ``dom0``, using the following command:
+
+.. code-block:: sh
+  
+  rm /tmp/{sd-journalist.sec journalist.txt}
+
+
+Uninstall SecureDrop Workstation
+--------------------------------
+To uninstall SecureDrop Workstation, open a ``dom0`` terminal and run the following command:
+
+.. code-block:: sh
+  
+  securedrop-admin --uninstall
```

Comment:
Instance secrets will remain in /usr/share/securedrop-workstation-dom0-config until manually deleted, per https://github.com/freedomofpress/securedrop-workstation/blob/cfceca0a7152efdf9f06bf8f0bfe0c9f8d95d689/scripts/securedrop-admin.py#L82

Reply:
Added a para addressing this.